tsconfig.json

```diff
@@ -12,7 +12,7 @@
       "forceConsistentCasingInFileNames": true,
       "noImplicitReturns": true,
       "noImplicitThis": true,
-      "noImplicitAny": true,
+      "noImplicitAny": false,
```

Comment:
I couldn't find some types, and this was breaking Storybook, so I turned it off.